include/mbedtls/mbedtls_config.h

```diff
@@ -3537,6 +3537,53 @@
  */
 //#define MBEDTLS_PSA_CRYPTO_USER_CONFIG_FILE "/dev/null"
 
+/**
```

Comment:
I do not think it should be added in `mbedtls_config.h`. 

I think it should be move to `psa_config.h` ( I am not sure the real name at this moment) or something like this.